lang/java/avro/src/main/java/org/apache/avro/io/parsing/Symbol.java

```diff
@@ -369,9 +371,19 @@ public Repeater flatten(Map<Sequence, Sequence> map,
    * for some inputs.
    */
   public static boolean hasErrors(Symbol symbol) {
+    return hasErrors(symbol, new HashSet<Symbol>());
+  }
+  
```

Comment:
Please remove trailing whitespaces from here, Checkstyle plugin doesn't like it, and mvn test fails